recipes/pistache/all/conanfile.py

```diff
@@ -0,0 +1,112 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.29.1"
+
+
+class PistacheConan(ConanFile):
+    name = "pistache"
+    license = "Apache-2.0"
+    homepage = "https://github.com/pistacheio/pistache"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("http", "rest", "framework", "networking")
+    description = "Pistache is a modern and elegant HTTP and REST framework for C++"
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False], "with_ssl": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_ssl": True}
```

Comment:
```suggestion
    default_options = {"shared": False, "fPIC": True, "with_ssl": False}
```

https://github.com/pistacheio/pistache/blob/ae146fa773019118ae48ef48fa91ba8888de79aa/CMakeLists.txt#L31

Reply:
I know, I just wanted to give full dependencies instead, but it's not fair.

Reply:
Fixed, done